common/src/main/java/com/wynntils/features/inventory/EmeraldPouchFillArcFeature.java

```diff
@@ -37,7 +37,7 @@ public void onRenderHotbarSlot(HotbarSlotRenderEvent.Pre e) {
     }
 
     @SubscribeEvent
-    public void onRenderSlot(SlotRenderEvent.Pre e) {
+    public void onRenderSlot(SlotRenderEvent.Post e) {
```

Comment:
Doesn't seem to be required here but better to keep it the same in case it changes